### ISSUE TITLE
Show LICENSE information when available

### ISSUE THREAD

**Issue by tkw1536** [shurcooL/gtdo#59]
Many projects and packages are subject to licenses that require for a copy of the license to be included with every copy of the code. 
When available, a copy of the license should be shown or linked on the package page.

**Comment by dmitshur** [comment]
I agree this information would be helpful to expose, especially when viewing specific commits other than the latest one on the main branch (something that gtdo lets users do).

However, this information is readily available at https://pkg.go.dev, so I'd suggest using that website for this purpose (i.e., swap out `gotools.org` with `pkg.go.dev` in the URL bar, keeping the rest of the URL the same).
